src/main/java/com/pi4j/drivers/display/graphics/PixelFormat.java

```diff
@@ -4,7 +4,8 @@ public enum PixelFormat {
 
     RGB_444(4, 4, 4), // 12-bit color format with 4 bits for each color channel (red, green, blue)
     RGB_565(5, 6, 5), // 16-bit color format that uses 5 bits for red, 6 bits for green, and 5 bits for blue
-    RGB_888(8, 8, 8);
+    RGB_565_LE(5, 6, 5), // The same as RGB_565, but the bytes swapped to little endian format.
```

Comment:
extend the definition to RGB_565_LE(5, 6, 5, true)

add 
```
private final boolean littleEndian

PixelFormat(int redBitCount, int greenBitCount, int blueBitCount) {
this(redBitCount, greenBitCount, blueBitCount, false);
}

PixelFormat(int redBitCount, int greenBitCount, int blueBitCount, boolean littleEndian) {
this.littleEndian = littleEndian;
... existing code
}

Reply:
I think this would be misleading: It would look like one could just add this parameter when in fact LE is only supported for RGB565. And it you think about it, it doesn't make much sense in other formats than RGB565. Byte aligned formats could just have a matching name instead. And what would it even mean for RGB444?